src/lib/backend/Memory.js

```diff
@@ -311,6 +313,18 @@ export default class Memory {
     }
 
     /**
+    * publish a message to a channel, clients can subsribe to this channel
+    * to receive messages
+    * @param {string} channel - channel to push the message
+    * @param {string} message - message to be pushed to the channel
+    * @return {object} - current instance
+    */
+    publish(channel, message) {
+        this.channels.emit('utapi', message);
```

Comment:
As it stands, with `memBackend`, the channel will always be `'utapi'`. To eventually accommodate multiple channels, shouldn't the `channel` parameter be passed to `this.channels.emit`?